src/math/fib.jl

```diff
@@ -0,0 +1,31 @@
+"""
+Program for calculating the nth fibonacci number
```

Comment:
Please add a source or Wikipedia link for algorithm explanation.
You can also add a link to your GitHub profile at the bottom like

```markdown
Contributed By:- [Your name](Link to your GitHub profile)
```